lib/cib/cib_file.c

```diff
@@ -397,7 +397,7 @@ load_file_cib(const char *filename, xmlNode **output)
     if (validate_xml(root, NULL, TRUE) == FALSE) {
         const char *schema = crm_element_value(root, PCMK_XA_VALIDATE_WITH);
 
-        crm_err("CIB does not validate against %s", schema);
+        crm_err("CIB does not validate against %s, or schema file missing", schema);
```

Comment:
I'd rather distinguish these cases for the user. Looking at what that might involve, it seems the `easy` tag was misplaced on this one. ;) Most likely T729 should be done first or in conjunction.

Reply:
Since that one is not a high priority, this one is a good idea in the meantime. Let's just edit this to say "... or that schema is unknown".